### ISSUE TITLE
Character Inventory suggestion

### ISSUE THREAD

**Issue by Adipa00** [Mantan21/Genshin-Impact-Wish-Simulator#98]
I think something that could help be accurate and helpfull is if there was a way to give yourself characters in the inventory that you already have in game and choose the constellation before pulling them. That way we have an easier way tracking constelation or characters owned and the pulling simulator reflects that. For instance, I alread have a c6 noelle in game, so in the simulator if I pull her it doesn't register her as a new character and gives the starglitter instead.

**Comment by AguzzTN54** [comment]
Yes, several users have already suggested this to me, and I have added some programmatical code about this. Maybe it won't take too long to wait for this feature to be ready.
